### ISSUE TITLE
Could not find bluetooth module; is the cinnamon-control-center package installed?

### ISSUE THREAD

**Issue by chris13524** [linuxmint/Cinnamon#5741]
I'm trying to access my power settings, however, when I go to them, the program crashes. When I run the command manually, it gives me this error:

``````
chris13524@lemon ~ % cinnamon-settings power
Python module
Could not find bluetooth module; is the cinnamon-control-center package installed?
Loading Power module
zsh: segmentation fault (core dumped)  cinnamon-settings power```
``````

**Comment by JosephMcc** [comment]
Do other modules work or is it just the power settings?

**Comment by chris13524** [comment]
Yes, it's just the power settings one. I do have an issue with my bluetooth, it seems I can't connect to anything, where my laptop is able to connect to the same device just fine.

**Comment by berislavlopac** [comment]
I'm experiencing the exact same issue, with the same error message as @chris13524. Just FYI I'm using an originally Ubuntu Gnome Xenial setup, updated to use the Cinnamon desktop.

**Comment by berislavlopac** [comment]
I'm experiencing the exact same issue, with pretty much the same error message as @chris13524. Just FYI I'm using an originally Ubuntu Gnome Xenial setup, updated to use the Cinnamon desktop.

This is the exact message I'm getting, in case it can help:

    berislav@shadowdance ~> cinnamon-settings power
    Python module
    Could not find bluetooth module; is the cinnamon-control-center package installed?
    Loading Power module
    fish: “cinnamon-settings power” terminated by signal SIGSEGV (Address boundary error)

**Comment by leigh123linux** [comment]
It was fixed in 3.0.7 release

https://github.com/linuxmint/Cinnamon/commit/7a97c28c652abe7a4e8a47787109e9ba9beeea51

**Comment by berislavlopac** [comment]
Ah I see -- thank you!
